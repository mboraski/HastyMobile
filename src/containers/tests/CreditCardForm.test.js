// TODO: Remove when tests implemented
test.only('real recognize real', () => {
    expect('real').toBe('real');
});
// import 'react-native';
// import React from 'react';
// import renderer from 'react-test-renderer';
// import { shallow } from 'enzyme';
// import configureStore from 'redux-mock-store';
//
// import '../../mocks/firebase';
// import { CreditCardForm } from '../CreditCardForm';
//
// describe('CreditCardForm', () => {
//     it('should render correctly', () => {
//         const props = {
//             navigation: { state: { params: {} } }
//         };
//         const render = shallow(<CreditCardForm {...props} />);
//         expect(render).toMatchSnapshot();
//     });
//     it('should render correctly with card', () => {
//         const props = {
//             navigation: { state: { params: { card: { last4: '4242' } } } }
//         };
//         const render = shallow(<CreditCardForm {...props} />);
//         expect(render).toMatchSnapshot();
//     });
// });
