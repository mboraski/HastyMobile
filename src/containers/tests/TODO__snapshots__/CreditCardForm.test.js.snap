// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreditCardForm should render correctly 1`] = `
<KeyboardAvoidingView
  behavior="padding"
  keyboardVerticalOffset={16}
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <Component
    style={
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        style={
          Object {
            "flexDirection": "row",
          }
        }
      >
        <CardNumberInputField
          containerStyle={
            Object {
              "flex": 1,
            }
          }
          keyboardType="number-pad"
          label="CARD NUMBER"
          name="number"
          normalize={[Function]}
          placeholder=""
          validate={[Function]}
        />
        <TextInputField
          containerStyle={
            Object {
              "width": 140,
            }
          }
          keyboardType="number-pad"
          label="EXP. DATE"
          name="exp"
          normalize={[Function]}
          style={
            Object {
              "textAlign": "center",
            }
          }
          validate={[Function]}
        />
      </View>
      <TextInputField
        label="CARDHOLDER NAME"
        name="name"
        validate={[Function]}
      />
      <TextInputField
        containerStyle={
          Object {
            "width": 120,
          }
        }
        keyboardType="number-pad"
        label="CVC"
        name="cvc"
        placeholder=""
        secureTextEntry={true}
        validate={[Function]}
      />
    </View>
  </Component>
</KeyboardAvoidingView>
`;

exports[`CreditCardForm should render correctly with card 1`] = `
<KeyboardAvoidingView
  behavior="padding"
  keyboardVerticalOffset={16}
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <Component
    style={
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        style={
          Object {
            "flexDirection": "row",
          }
        }
      >
        <CardNumberInputField
          containerStyle={
            Object {
              "flex": 1,
            }
          }
          keyboardType="number-pad"
          label="CARD NUMBER"
          name="number"
          normalize={[Function]}
          placeholder="**** **** **** 4242"
          validate={[Function]}
        />
        <TextInputField
          containerStyle={
            Object {
              "width": 140,
            }
          }
          keyboardType="number-pad"
          label="EXP. DATE"
          name="exp"
          normalize={[Function]}
          style={
            Object {
              "textAlign": "center",
            }
          }
          validate={[Function]}
        />
      </View>
      <TextInputField
        label="CARDHOLDER NAME"
        name="name"
        validate={[Function]}
      />
      <TextInputField
        containerStyle={
          Object {
            "width": 120,
          }
        }
        keyboardType="number-pad"
        label="CVC"
        name="cvc"
        placeholder="***"
        secureTextEntry={true}
        validate={[Function]}
      />
    </View>
    <Button
      buttonStyle={
        Object {
          "backgroundColor": "#fff",
          "justifyContent": "flex-start",
          "paddingHorizontal": 23.466666666666665,
          "paddingVertical": 15,
        }
      }
      containerViewStyle={
        Object {
          "borderBottomWidth": 0.5,
          "borderColor": "#cccccc",
          "borderTopWidth": 0.5,
          "marginBottom": 20,
          "marginLeft": 0,
          "marginRight": 0,
        }
      }
      onPress={[Function]}
      textStyle={
        Object {
          "color": "#000",
          "fontSize": 16,
        }
      }
      title="Delete Card"
    />
  </Component>
</KeyboardAvoidingView>
`;
